clipboard.go

```diff
@@ -5,13 +5,20 @@
 // Package clipboard read/write on clipboard
 package clipboard
 
+import "os"
+
+var isSsh = os.Getenv("SSH_TTY") != ""
```

Comment:
Conditioning on SSH seems strange. I see how it narrowly focuses it on your case, but is there a better way to tell OSC-52 should be used?